opensrp-app/src/main/java/org/smartregister/domain/ResponseErrorStatus.java

```diff
@@ -0,0 +1,20 @@
+package org.smartregister.domain;
+
+public enum ResponseErrorStatus {
+    malformed_url("Malformed Url"),
```

Comment:
Move this to the strings file to enable translations

Reply:
its an enum value